clang/test/SemaSYCL/reqd-sub-group-size-host.cpp

```diff
@@ -1,9 +1,53 @@
-// RUN: %clang_cc1 -fsycl -fsycl-is-host -fsyntax-only -verify %s
-// expected-no-diagnostics
+// RUN: %clang_cc1 -fsycl -fsycl-is-host -Wno-sycl-2017-compat -fsyntax-only -verify %s
```

Comment:
I don't think you need 2 runs here you can add -ast-dump to first run.

Reply:
Fixed that.